lib/queue.js

```diff
@@ -1,27 +1,33 @@
 class Queue {
     constructor() {
```

Comment:
👀 While your implementation does work and utilize Javascript arrays effectively, the readme asks you to implement a queue using a circular buffer.